internal/storage/bucket_handle.go

```diff
@@ -471,28 +471,37 @@ func (b *bucketHandle) DeleteFolder(ctx context.Context, folderName string) (err
 	var callOptions []gax.CallOption
 
 	err = b.controlClient.DeleteFolder(ctx, &controlpb.DeleteFolderRequest{
-		Name: "projects/_/buckets/" + b.bucketName + "/folders/" + folderName,
+		Name: fmt.Sprintf("projects/_/buckets/%s/folders/%s", b.bucketName, folderName),
```

Comment:
This is used at two places. please create a constant.

Reply:
you need to create a constant for projects/_/buckets/%s/folders/%s and then use it in fmt.sprintf